plugins/GSdx/GLLoader.cpp

```diff
@@ -276,9 +276,9 @@ namespace GLLoader {
 			mesa_amd_buggy_driver = intel_buggy_driver = true;
 
 #ifdef _WIN32
-		buggy_sso_dual_src = intel_buggy_driver || fglrx_buggy_driver || legacy_fglrx_buggy_driver;
+		buggy_sso_dual_src = intel_buggy_driver || legacy_fglrx_buggy_driver;
```

Comment:
Unfortunately, GSdx isn't a testsuite to validate driver so I only check the string name. Therefore all version of the AMD driver will set `fglrx_buggy_driver`.  It was kind of working on previous AMD driver (PCSX2 not the testcase). I guess most people update their drivers regularly to enjoy new video game.